plugin/Record/RecordManagerDuplicateListViewForm.php

```diff
@@ -117,20 +120,8 @@ public function duplicateByName(string $name)
 				foreach ($listViewRoot['fields'] as $field) {
 					$selectFieldConfigByIdAndFieldName = "SELECT * FROM {$this->_pWPDB->_escape($tableFieldConfig)} WHERE form_id='{$this->_pWPDB->_escape($id)}' AND fieldname ='{$this->_pWPDB->_escape($field)}'";
 					$fieldConfigRows = $this->_pWPDB->get_results($selectFieldConfigByIdAndFieldName);
```

Comment:
Please use `get_results()` with the 2nd parameter `ARRAY_A`, so that the code becomes easier. Especially the method `duplicateDataRelated`